common/dna_data/wc_orc_dna.txt

```diff
@@ -0,0 +1,1195 @@
+ 		gene_bs_nose_ridge_width={ "nose_ridge_width_neg" 29 "nose_ridge_width_pos" 203 }
+ 		gene_bs_nose_size={ "nose_size_neg" 8 "nose_size_neg" 8 }
+ 		gene_bs_nose_tip_angle={ "nose_tip_angle_pos" 121 "nose_tip_angle_pos" 197 }
+ 		gene_bs_nose_tip_forward={ "nose_tip_forward_neg" 121 "nose_tip_forward_neg" 193 }
+ 		gene_bs_nose_tip_width={ "nose_tip_width_pos" 205 "nose_tip_width_neg" 89 }
+ 		face_detail_cheek_def={ "cheek_def_02" 255 "cheek_def_02" 175 }
+ 		face_detail_cheek_fat={ "cheek_fat_01_neg" 11 "cheek_fat_01_neg" 6 }
+ 		face_detail_chin_cleft={ "chin_dimple" 16 "chin_dimple" 16 }
+ 		face_detail_chin_def={ "chin_def" 13 "chin_def" 13 }
+ 		face_detail_eye_lower_lid_def={ "eye_lower_lid_def" 255 "eye_lower_lid_def" 243 }
+ 		face_detail_eye_socket={ "eye_socket_03" 253 "eye_socket_03" 137 }
+ 		face_detail_nasolabial={ "nasolabial_03" 255 "nasolabial_03" 255 }
+ 		face_detail_nose_ridge_def={ "nose_ridge_def_pos" 70 "nose_ridge_def_neg" 0 }
+ 		face_detail_nose_tip_def={ "nose_tip_def" 164 "nose_tip_def" 164 }
+ 		face_detail_temple_def={ "temple_def" 255 "temple_def" 161 }
+ 		expression_brow_wrinkles={ "brow_wrinkles_04" 255 "brow_wrinkles_01" 235 }
+ 		expression_eye_wrinkles={ "eye_wrinkles_03" 240 "eye_wrinkles_01" 240 }
+ 		expression_forehead_wrinkles={ "forehead_wrinkles_01" 255 "forehead_wrinkles_01" 191 }
+ 		expression_other={ "nose_wrinkles_01" 245 "cheek_wrinkles_both_01" 245 }
+ 		complexion={ "complexion_5" 219 "complexion_5" 219 }
+ 		gene_height={ "normal_height" 170 "normal_height" 163 }
+ 		gene_bs_body_type={ "body_fat_head_fat_medium" 155 "body_fat_head_fat_low" 155 }
+ 		gene_bs_body_shape={ "body_shape_average_clothed" 255 "body_shape_pear_half" 217 }
+ 		gene_bs_bust={ "bust_clothes" 90 "bust_shape_2_full" 90 }
+ 		gene_age={ "old_3" 255 "old_3" 87 }
+ 		gene_eyebrows_shape={ "avg_spacing_lower_thickness" 130 "avg_spacing_avg_thickness" 117 }
+ 		gene_eyebrows_fullness={ "layer_2_lower_thickness" 171 "layer_2_lower_thickness" 132 }
+ 		gene_body_hair={ "body_hair_sparse" 133 "body_hair_sparse" 126 }
+ 		hairstyles={ "northern_hairstyles" 191 "orcish_hairstyles" 24 }
+ 		beards={ "northern_beards" 135 "orcish_beards" 77 }
+ 		eye_accessory={ "normal_eyes" 46 "normal_eyes" 46 }
+ 		teeth_accessory={ "normal_teeth" 0 "normal_teeth" 0 }
+ 		eyelashes_accessory={ "no_eyelashes" 222 "no_eyelashes" 222 }
+ 		gene_bs_ear_lenght={ "erect_ear_lenght" 29 "erect_ear_lenght" 29 }
+ 		special_eyebrows={ "no_eyebrows" 111 "no_eyebrows" 111 }
+ 		special_eyes={ "no_special_eyes" 255 "no_special_eyes" 0 }
+ 		gene_bs_mouth_tusks={ "mouth_tusks_pos" 255 "mouth_tusks_pos" 255 }
+ 		static_tusks={ "orcish_tusks" 127 "orcish_tusks" 193 }
+ 		gene_bs_ear_tip={ "ear_tip_pos" 255 "ear_tip_pos" 255 }
+            gene_skin_value={ "skin_value_neg" 0 "skin_value_neg" 0 }
+		}
+	}
+	enabled=yes
+}
+
+blackhand_dna_2 = { # base by Garthor13, adjusted by ercarp
```

Comment:
He has no hairs and beard. Intented?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114060581-92cf8a00-98a6-11eb-9af5-46d8fbf5a7f9.png)

</details>

Reply:
Not intended. Fixed in ed5cd8f2efd73481cb3dcf379af77f208a2ef5df.